google/cloud/storage/benchmarks/storage_throughput_vs_cpu_benchmark.cc

```diff
@@ -268,6 +285,9 @@ TestResults RunThread(Options const& options, std::string const& bucket_name) {
   std::bernoulli_distribution crc_generator;
   std::bernoulli_distribution md5_generator;
 
+  std::uniform_int_distribution<std::size_t> api_generator(
+      0, options.enabled_apis.size() - 1);
```

Comment:
Should we have checked `!options.enabled_apis.empty()` somewhere?

Reply:
That is done in `ParseArgsDefault()` along with all other argument validation.

Reply:
Oops ... I see it now.  Thanks.